CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -229,12 +258,17 @@ class SchedulingPlacesViewController: UIViewController {
             make.width.equalTo(14)
             make.height.equalTo(24)
         }
-
     }
 
     // MARK: Button Action
     private func updateNext() {
-        let enable = selectedCtownLocations.count + selectedCampusLocations.count > 2 || pickedLocation != nil
+        let enable: Bool
```

Comment:
Again can we say `let enable = schedulingStatus == .choosing ? pickedLocation != nil  : totalSelectedLocations > 2`?